cli/src/commands/start.rs

```diff
@@ -27,6 +27,19 @@ use rand_chacha::ChaChaRng;
 use std::net::SocketAddr;
 use tokio::runtime::{self, Runtime};
 
+/// Recommended minimum value of 'open files' limit for beacon node.
+///
+/// Beacon should be able to handle at least 1000 concurrent connections,
+/// each requiring 2 sockets.
+#[cfg(target_family = "unix")]
+const RECOMMENDED_MIN_NOFILES_LIMIT_BEACON: u64 = 2048;
+/// Recommended minimum value of 'open files' limit for validator node.
+///
+/// Beacon should be able to handle at least 500 concurrent connections,
+/// each requiring 2 sockets.
+#[cfg(target_family = "unix")]
+const RECOMMENDED_MIN_NOFILES_LIMIT_VALIDATOR: u64 = 1024;
```

Comment:
@howardwu @niklaslong please help choosing correct values for those constants :)